media-library/overview/copy-and-move-folders.md

```diff
@@ -17,6 +17,20 @@ ImageKit.io provides users with the ability to copy and move folders across diff
 
 ![New destination for copying/moving the selected folders](<../../.gitbook/assets/copy-move-folder-drawer.png>)
 
+## Access control and permissions
+
+## Copy folder
+A restricted media library user needs to have at least "contribute" permission on both the folder and destination folder to be able to do this action.
+
+On completion of a copy folder operation, access control and permission levels for the source folder remain unchanged. If a folder at the destination location with the same name exists, the ownership, access control, and permissions present at the destination location's folder would persist. If no folder exists in the destination folder, the copied asset's ownership is set to the user who did the copy operation. Additionally, the destination location’s permissions from shared parent folders would come into effect.
+
+## Move folder
+A restricted media library user needs to have at least "manage" permission on both the folder and destination folder to be able to do this action.
+
+If a folder at the destination location with the same name exists, new folder versions would be created, and the ownership, access control, and permissions present at the destination location's folder would persist. If the source folder was added to any media collection, the destination file would also now be added to those media collections along with any existing media collections it was added to. If no file exists in the destination folder, the moved asset's ownership, access control, and permissions remain unchanged. Additionally, the destination location’s permissions from shared parent folders would come into effect.
```

Comment:
This para needs to be rewritten. Folder versions do not exist. Folders will be merged together.

> the destination file would also now be added to those media collections

destination folder.

Reply:
done